vc/vc.go

```diff
@@ -190,12 +192,27 @@ func (vc VerifiableCredential) Proofs() ([]Proof, error) {
 }
 
 func (vc VerifiableCredential) MarshalJSON() ([]byte, error) {
+	if vc.raw != "" {
+		// Credential instance created through ParseVerifiableCredential()
+		if vc.format == JWTCredentialProofFormat {
+			// Marshal as JSON string
+			return json.Marshal(vc.raw)
+		}
+		// JSON-LD, already in JSON format so return as-is
+		return []byte(vc.raw), nil
+	}
+	// Must be a (new) JSON-LD credential (library does not support creating JWT VCs)
 	type alias VerifiableCredential
 	tmp := alias(vc)
 	if data, err := json.Marshal(tmp); err != nil {
 		return nil, err
 	} else {
-		return marshal.NormalizeDocument(data, pluralContext, marshal.Unplural(typeKey), marshal.Unplural(credentialSubjectKey), marshal.Unplural(proofKey))
+		return marshal.NormalizeDocument(data, pluralContext, marshal.Unplural(typeKey), marshal.Unplural(credentialSubjectKey), marshal.Unplural(proofKey),
```

Comment:
is this on the correct type? Shouldn't the JSON-LD be unchanged and the JWT-VC skip these fields when they are zero?

Reply:
Reverted this, instead introduced `CreateJWTVerifiableCredential()` to avoid creating a JWT VC with un-initialized fields due to them being present as JWT claims